new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { getColorClass, productLinks } from '@/utils/memberUtils';
+import Image from 'next/image';
+import Link from 'next/link';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}`}
+      className="rounded-lg w-auto h-auto"
+      width={232}
+      height={232}
+    />
+    <div className="text-left">
+      {enlarged ? (
+        <h3>{`${user.firstName} ${user.lastName}`}</h3>
+      ) : (
+        <h6>{`${user.firstName} ${user.lastName}`}</h6>
+      )}
+      <p className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</p>
+    </div>
+  </>
+);
+
+type MemberCardProps = {
+  user: IdolMember;
+  image: string;
+  selected: boolean;
+  onClick?: () => void;
+};
+
+export const MemberCard = ({ user, image, selected, onClick }: MemberCardProps) => {
+  const baseStyles =
+    'p-8 border border-border-1 flex flex-col gap-4 focusState hover:shadow-white hover:shadow-[0_-4px_8px_0]';
+
+  return (
+    <button
+      className={`${baseStyles} ${selected ? 'border-b-foreground-1' : ''}`}
```

Comment:
```suggestion
      className={`${baseStyles} ${selected ? 'bg-background-2 border-b-foreground-1' : ''}`}
```

selected state should have slightly elevated background color

Reply:
Also can we add a shadow like the one on the navbar for the currently selected card:

<img width="157" alt="Screenshot 2025-05-13 at 4 34 54 PM" src="https://github.com/user-attachments/assets/dc159507-ffc3-4afa-9720-5352a7c12db2" />


I think the way [I implemented it for the navbar](https://vscode.dev/github/cornell-dti/idol/blob/cc/team-card/new-dti-website-redesign/src/components/Navbar.tsx#L49-L50) was by using an `::after` pseudo-element and then some custom shadow properties